achievements/achievements.ts

```diff
@@ -2509,14 +2626,32 @@ const achievements: Achievement[] = [
 		value: 1,
 	},
 	{
-		id: 'kirafan-answer-first-hint-5',
+		id: 'kirafan-answer-first-hint-5', // XXX
 		difficulty: 'professional',
 		title: '女神候補生',
 		condition: 'きらファン当てクイズでヒント1回以内で10回正解する',
 		category: 'kirafan',
 		counter: 'kirafan-answer-first-hint',
 		value: 10,
 	},
+	{
+		id: 'kirafan-answer-first-hint-100',
+		difficulty: 'professional',
+		title: '筆頭神官',
+		condition: 'きらファン当てクイズでヒント1回以内で100回正解する',
+		category: 'kirafan',
+		counter: 'kirafan-answer-first-hint',
+		value: 100,
+	},
+	{
+		id: 'kirafan-answer-first-hint-1000',
+		difficulty: 'professional',
+		title: '女神ソラ',
+		condition: 'きらファン当てクイズでヒント1回以内で1000回正解する',
+		category: 'kirafan',
+		counter: 'kirafan-answer-first-hint',
+		value: 1000,
+	},
```

Comment:
```suggestion
	{
		id: 'kirafan-answer-first-hint-100',
		difficulty: 'professional',
		title: 'きららさんといっしょ',
		condition: 'きらファン当てクイズでヒント1回以内で100回正解する',
		category: 'kirafan',
		counter: 'kirafan-answer-first-hint',
		value: 100,
	},
	{
		id: 'kirafan-answer-first-hint-1000',
		difficulty: 'professional',
		title: '新たなる聖典の紡ぎ手',
		condition: 'きらファン当てクイズでヒント1回以内で1000回正解する',
		category: 'kirafan',
		counter: 'kirafan-answer-first-hint',
		value: 1000,
	},
```
きらファン当てクイズヒント1回の実績はきららファンタジアオリジナルキャラクターであるところのランプさんを題材にしているので、実績名はこちらのほうが嬉しい